### ISSUE TITLE
qgc crash

### ISSUE THREAD

**Issue by waterflyfish** [mavlink/qgroundcontrol#7095]
Hi, we have a severe problem with video display. When video transmission is interrupted, e.g. wifi is broken, or video transmission is lost due to long distance, the ground station would freeze for about 30 seconds. If the user continues to try to operate, the ground station would crash altogether. We used Qt v5.11 to compile. Please help asap. Thanks.

**Comment by jaxxzer** [comment]
Can you run the application in debug mode and provide a stack trace/application output when the crash is encountered? Which commit are you using?

It is impossible to help you with so little information about the problem.

**Comment by waterflyfish** [comment]
Hi,jaxxzer,thanks for your reply.We ran the program in debug mode and found that when video was disconnected due to network signal, the program would block in the processing method of timer _frameTimer (). Oddly, the program does not continue, but qt does not prompt any error messages. The QGC source code version we used was 3.4.3. By the way, the qgroundcontrol-installer. exe we downloaded from git also has this problem.Thanks.
